stack.yaml

```diff
@@ -70,3 +70,5 @@ ghc-options:
 
  # See https://github.com/haskell/haskell-language-server/issues/208
  "$everything": -haddock
+
+ statistics: -fsimpl-tick-factor=10000 # statistics fails on GHC 9 without this, https://github.com/haskell/statistics/issues/173
```

Comment:
Known issue with the statistics package that makes it fail to build on `stack build --profile` runs, this fixes it, can probably remove this in a future snapshot if they sort it out on the package side.